types/lib/elements.d.ts

```diff
@@ -60,12 +60,14 @@ export interface CardElement extends Emitter<ElementEvent> {
   attach: Attach<CardElement>;
   remove: () => CardElement;
   configure: (options: CardElementOptions) => CardElement;
+  focus: () => CardElement;
 }
 
 export interface IndividualElement extends Emitter<ElementEvent> {
   attach: Attach<IndividualElement>;
-  remove: () => Element;
```

Comment:
`Element` here was referring to https://developer.mozilla.org/en-US/docs/Web/API/Element